next-app/hooks/useClusterColor.ts

```diff
@@ -1,30 +1,30 @@
 import { useCallback, useMemo } from "react";
 
 const colors: string[] = [
+  "#4169e1", // Royal Blue
```

Comment:
Thank you so much for improving the colors! 🙏 
The list I had put there was generated with chatpgt and it wasn't great.